src/components/Header/Header.tsx

```diff
@@ -5,25 +5,41 @@ export interface HeaderProps {
   left: ReactNode;
   center: ReactNode;
   right: ReactNode;
+  centerIsShown?: boolean;
 }
 
-export const Header: React.FC<HeaderProps> = ({ left, center, right }: HeaderProps) => (
+export const Header: React.FC<HeaderProps> = ({
+  left,
+  center,
+  right,
+  centerIsShown = true,
```

Comment:
By convention, boolean arguments should default to false.
By defaulting to false, the argument can be set by including or excluding it from the JSX:
```tsx
<Header hideCenter> // hideCenter === true
<Header> // hideCenter === false
```

```suggestion
  hideCenter = false,
```